karma.conf.js

```diff
@@ -89,7 +89,15 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['Chrome_Sized'],
+
+
+    customLaunchers: {
+      'Chrome_Sized': {
+        base: 'Chrome',
+        flags: ['--window-size=400,600']
```

Comment:
@misscs It does - I just couldn't get it to work. Let me try again though because I do agree that would be the best way to go.